ui/main/src/app/business/services/realtime-domain.service.ts

```
@@ -128,6 +133,10 @@
             endDomain = endOfWeekTime;
         }
 
+        if (this.currentDomainId === 'Unsynchronised') {
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)